src/bci_build/package/rust/README.md.j2

```diff
@@ -0,0 +1,48 @@
+# {{ image.pretty_name }} Container Image
+
+{% include 'badges.j2' %}
+
+## Description
+
+[Rust](https://www.rust-lang.org/) is a systems programming language sponsored by Mozilla Research. It is designed to be a "safe, concurrent, practical language", supporting functional and imperative-procedural paradigms. Rust is syntactically similar to C++, but is designed for better memory safety while maintaining performance.
+
+## Usage
+
+To compile and deploy an application, copy the sources, fetch dependencies, and build the binary:
+
+```Dockerfile
+# Build the application using the {{ image.pretty_name }} Container Image
+FROM {{ image.pretty_reference }} as build
+
+WORKDIR /app
+
+COPY . ./
+
+RUN cargo build --release
+
+# Bundle the application into a BCI minimal (or other BCI image)
+FROM registry.suse.com/bci/bci-minimal
```

Comment:
Maybe suggest `micro` instead? `minimal` is `micro` + `rpm`, but we're not leveraging `rpm` in this example.
```suggestion
FROM registry.suse.com/bci/bci-micro
```

Reply:
Changed.